indy_node/utils/node_control_tool.py

```diff
@@ -45,7 +42,7 @@ def __init__(
         self.config = config or getConfig()
 
         assert self.config.UPGRADE_ENTRY, "UPGRADE_ENTRY config parameter must be set"
-        self.ext_upgrade = self.config.UPGRADE_ENTRY not in PACKAGES_TO_HOLD
+        self.upgrade_entry = self.config.UPGRADE_ENTRY
```

Comment:
Why is it always from config? Why not from POOL_UPGRADE command?

Reply:
this is initialization stage. proper name will be provided by the node via upgrade message